src/Core/Settings.cpp

```diff
@@ -1794,7 +1794,7 @@ Possible values:
 
 - 0 — Disabled.
 - 1 — Enabled.
-)", 0) \
+)", 1) \
```

Comment:
This change was incorrect and should have changed the first line (false -> true). Reverting in https://github.com/ClickHouse/ClickHouse/pull/72319 since it requires test changes and we need to keep things clean for 24.11. Need to be reintroduced with those fixes for 24.12